engine/src/main/java/org/terasology/engine/world/generation/facets/base/BaseFacet3D.java

```diff
@@ -32,13 +32,15 @@ protected final int getRelativeIndex(int x, int y, int z) {
         if (!relativeRegion.contains(x, y, z)) {
             throw new IllegalArgumentException(String.format("Out of bounds: (%d, %d, %d) for region %s", x, y, z, relativeRegion.toString()));
         }
-        return x - relativeRegion.minX() + relativeRegion.getSizeX() * (y - relativeRegion.minY() + relativeRegion.getSizeY() * (z - relativeRegion.minZ()));
+        return x - relativeRegion.minX() + relativeRegion.getSizeX()
+                * (y - relativeRegion.minY() + relativeRegion.getSizeY() * (z - relativeRegion.minZ()));
```

Comment:
Does this fit into one line?
```suggestion
        return x - relativeRegion.minX() 
                + relativeRegion.getSizeX() * (y - relativeRegion.minY() + relativeRegion.getSizeY() * (z - relativeRegion.minZ()));
```